src/middlewared/middlewared/environ.py

```diff
@@ -0,0 +1,19 @@
+import os
```

Comment:
How about we put it inside utils? or common?

Dont feel strongly about it if you dont think its necessary, jsut not sure if I would fit this into the middlewared level of things.